src/allmydata/test/test_storage.py

```diff
@@ -1891,10 +1900,10 @@ def test_remove_non_present(self):
         )
 
         self.assertTrue(testv_is_good)
-        self.assertEqual({}, read_data)
+        self.assertThat({}, Equals(read_data))
 
 
-class MDMFProxies(unittest.TestCase, ShouldFailMixin):
+class MDMFProxies(SyncTestCase, ShouldFailMixin):
```

Comment:
setUp and tearDown need to call the super implementations

Reply:
Resolved by calling super implementations